test/integration/test_utils.mjs

```diff
@@ -176,6 +176,28 @@ async function getFirstSerialized(page, filter = undefined) {
   return (await getSerialized(page, filter))[0];
 }
 
+function getAnnotationStorage(page) {
+  return page.evaluate(() =>
+    Object.fromEntries(
+      window.PDFViewerApplication.pdfDocument.annotationStorage.serializable.map.entries()
```

Comment:
In the general case this could throw, since `serializable.map` could be `null`; note the default values in https://github.com/mozilla/pdf.js/blob/7b89e7e0d2d14fce7521fcc084c24827990ecb0b/src/display/annotation_storage.js#L20-L24